packages/widgets/src/blocks/widget-group/edit.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	useBlockProps,
+	BlockIcon,
+	ButtonBlockAppender,
+	InnerBlocks,
+	store as blockEditorStore,
+} from '@wordpress/block-editor';
+import { Placeholder, TextControl } from '@wordpress/components';
+import { group as groupIcon } from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+import { useSelect } from '@wordpress/data';
+import { getBlockType } from '@wordpress/blocks';
+
+export default function Edit( props ) {
+	const { clientId, isSelected } = props;
+	const { getBlock } = useSelect( blockEditorStore );
+	const { innerBlocks } = getBlock( clientId );
```

Comment:
Probably not the best idea to use selector during render phase directly 😅. From https://github.com/WordPress/gutenberg/blob/trunk/packages/data/README.md#useselect:

> Don't use `useSelect` this way when calling the selectors in the render function because your component won't re-render on a data change.

Reply:
Oops, brain fart. Fixed in 0e4e16f772.